samples/host_verify/host.c

```diff
@@ -202,8 +252,8 @@ void print_syntax(const char* program_name)
 {
     fprintf(
         stdout,
-        "Usage:\n  %s -r <report_file> [-e <endorsement_file>]\n  %s -c "
-        "<certificate_file>\n",
+        "Usage:\n  %s -r <report_file> -v <evidence_file> "
```

Comment:
Can you also update the samples/host_verify/README.md for changes in the command line and expected usage?

@yentsanglee can you also review it from the attestation samples perspective?

Reply:
Sure, README has been updated.

Reply:
Endorsement_file is an optional for -r and -v:

        "Usage:\n  %s -r <report_file> [-e <endorsement_file>]\n"
                      "  %s -v <evidence_file> "[-e <endorsement_file>]\n"
                      "  %s -c <certificate_file>\n",
                      program_name,
                      program_name,
                      program_name);

Reply:
Done, thanks!